### DISCUSSION TITLE
What is the maximum size for uploading firmware to LVFS

### DISCUSSION THREAD

**Discussion by Jierger** [fwupd/fwupd#3266] [Q&A] [answered]
The product is importing the fwupd update, I tried to upload the firmware, and found that the maximum size is only 100M. Is this limit standard? Could the size be opened larger? 

Thanks much.

**Comment by hughsie** [comment, marked as answer]
It is 100MB due to resource constraints. What exactly are you trying to upload? How much bigger than 100MB is it?
